src/utils/graph.ts

```diff
@@ -136,6 +136,8 @@ export const formatGraphQuery = <E extends EntityKey, K extends EntityKeys<E>>(
     args += (args.length ? ', ' : '') + `${name}: ` + value
   }
 
+  if (!opts.keys.includes('id' as K)) opts.keys.push('id' as K)
```

Comment:
i can't remember why `id` was ever hard-coded into the query string, and i don't think it needs to be. can we just remove this line for simplicity?